### PR TITLE
Update win11-overview.md

### PR DESCRIPTION
**Pull request by sh783**
@VinnieFlynt @dougeby for review. Removed references to pre-release Windows 11 on main overview.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/managed-desktop/intro/win11-overview.md [hunk 1]
```diff
@@ -17,40 +17,38 @@ Following the announcement of Windows 11, you might have started planning Window
...
+Windows 11 became generally available on October 4, 2021. It is ready for consumer and enterprise deployment and is a fully supported platform. We will begin scheduling deployments for all Microsoft Managed Desktop devices starting January 2023, but will provide full support for those that wish to deploy Windows 11 sooner. We'll consult and advise admins to develop and implement migration plans for each tenant based on technical readiness and your business considerations.
 
-You are welcome to install preview builds on devices whether they are managed by Microsoft Managed Desktop or not. We’ll continue to support Windows 10 in parallel until it reaches end of enterprise support. Please see [Windows 10 release information](/windows/release-health/release-information) for life cycle information.
+Microsoft Managed Desktop continues to support Windows 10 in parallel until it reaches end of enterprise support. Please see [Windows 10 release information](/windows/release-health/release-information) for life cycle information.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/intro/win11-overview.md]:
```suggestion
Microsoft Managed Desktop continues to support Windows 10 in parallel until it reaches end of enterprise support. See [Windows 10 release information](/windows/release-health/release-information) for life cycle information.
```

File: microsoft-365/managed-desktop/intro/win11-overview.md [hunk 2]
```diff
@@ -17,40 +17,38 @@ Following the announcement of Windows 11, you might have started planning Window
...
 
-For Microsoft Managed Desktop devices, you can request to add test devices to the **Modern Workplace - Windows 11 Pre-Release Test Devices** device group. This group receives Windows 11 preview builds along with a Microsoft Managed Desktop baseline configuration. Microsoft Managed Desktop doesn't manage the release cadence of Windows 11 preview builds, so members of this device group might receive updates more frequently than Windows 10 device groups.
+For Microsoft Managed Desktop devices, you can [add devices to the Windows 11 test device group] (https://docs.microsoft.com/en-us/microsoft-365/managed-desktop/working-with-managed-desktop/test-win11-mmd?view=o365-worldwide#add-devices-to-the-windows-11-test-group). This group receives the Windows 11 general availability build along with a Microsoft Managed Desktop baseline configuration. Once added to the device group, allow 1-2 days for a device to pick up the new settings and be offered Windows 11.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/intro/win11-overview.md]:
```suggestion
For Microsoft Managed Desktop devices, you can [add devices to the Windows 11 test device group](/microsoft-365/managed-desktop/working-with-managed-desktop/test-win11-mmd?view=o365-worldwide#add-devices-to-the-windows-11-test-group). This group receives the Windows 11 general availability build along with a Microsoft Managed Desktop baseline configuration. Once added to the device group, allow one to two days for a device to pick up the new settings and be offered Windows 11.
```

File: microsoft-365/managed-desktop/intro/win11-overview.md [hunk 3]
```diff
@@ -17,40 +17,38 @@ Following the announcement of Windows 11, you might have started planning Window
...
 
-For your devices that aren't managed by Microsoft Managed Desktop, you can join the [Windows Insider Program](/windows-insider/) to download preview builds and get guidance on deploying Windows 11 yourself. If you have devices running Windows 11 pre-release builds and later enroll them in Microsoft Managed Desktop, they won't revert back to Windows 10.
+For your devices that aren't managed by Microsoft Managed Desktop, you can read [Endpoint Manager guidance](https://techcommunity.microsoft.com/t5/microsoft-endpoint-manager-blog/endpoint-manager-simplifies-upgrades-to-windows-11/ba-p/2771886) to learn about deploying Windows 11 yourself. If you have devices running Windows 11 and later enroll them in Microsoft Managed Desktop, they won't revert back to Windows 10.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/intro/win11-overview.md]:
```suggestion
For your devices that aren't managed by Microsoft Managed Desktop, you can read [Endpoint Manager guidance](https://techcommunity.microsoft.com/t5/microsoft-endpoint-manager-blog/endpoint-manager-simplifies-upgrades-to-windows-11/ba-p/2771886) to learn about deploying Windows 11 yourself. If you have devices running Windows 11 and later, enroll them in Microsoft Managed Desktop; they won't revert back to Windows 10.
```

File: microsoft-365/managed-desktop/intro/win11-overview.md [hunk 4]
```diff
@@ -17,40 +17,38 @@ Following the announcement of Windows 11, you might have started planning Window
...
 
 ### Reactive measures
-If you encounter app compatibility issues in test or production environments, you can receive no-cost support by engaging [App Assure](/fasttrack/products-and-capabilities#app-assure) or FastTrack, as appropriate. For Windows 11, this includes any functionality with Office, Microsoft Edge, Teams, and line-of-business applications running on the latest operating system builds. App Assure directly engages app publishers to prioritize and resolve app compatibility issues.
+If you encounter app compatibility issues in test or production environments, you can receive no-cost support by opening a [service request] (https://docs.microsoft.com/en-us/microsoft-365/managed-desktop/working-with-managed-desktop/test-win11-mmd?view=o365-worldwide#report-issues). For Windows 11, this includes any functionality with Office, Microsoft Edge, Teams, and line-of-business applications running on the latest operating system builds. Microsoft App Assure directly engages app publishers to prioritize and resolve app compatibility issues when needed.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/intro/win11-overview.md]:
```suggestion
If you encounter app compatibility issues in test or production environments, you can receive no-cost support by opening a [service request](/microsoft-365/managed-desktop/working-with-managed-desktop/test-win11-mmd?view=o365-worldwide#report-issues). For Windows 11, this includes any functionality with Office, Microsoft Edge, Teams, and line-of-business applications running on the latest operating system builds. Microsoft App Assure directly engages app publishers to prioritize and resolve app compatibility issues when needed.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@sh783 Please implement the suggested changes. Thanks

**Pull request merged**